pyodide-build/pyodide_build/pywasmcross.py

```diff
@@ -534,42 +444,35 @@ def handle_command_generate_args(  # noqa: C901
 
     build_args The arguments that pywasmcross was invoked with
 
-    is_link_command Is this a linker invocation?
-
     Returns
     -------
         An updated argument list suitable for use with emscripten.
 
-
     Examples
     --------
 
     >>> from collections import namedtuple
     >>> Args = namedtuple('args', ['cflags', 'cxxflags', 'ldflags', 'target_install_dir'])
     >>> args = Args(cflags='', cxxflags='', ldflags='', target_install_dir='')
-    >>> handle_command_generate_args(['gcc', 'test.c'], args, False)
+    >>> handle_command_generate_args(['gcc', 'test.c'], args)
     ['emcc', 'test.c', '-Werror=implicit-function-declaration', '-Werror=mismatched-parameter-types', '-Werror=return-type']
     """
     if "-print-multiarch" in line:
         return ["echo", "wasm32-emscripten"]
-    for arg in line:
-        if arg.startswith("-print-file-name"):
-            return line
```

Comment:
Why don't we need this anymore?

Reply:
emcc now handles this flag, so I think we don't need this anymore:

https://github.com/emscripten-core/emscripten/blob/ec2ce4ce1d1596e3c45bdcd0a8e7cdeb0fe4ab3f/emcc.py#L609